tests/integration/test_http_transport.py

```
@@ -0,0 +1,169 @@
+            sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            sock.settimeout(0.1)
+            result = sock.connect_ex(("127.0.0.1", 8765))
+            sock.close()
+            if result == 0:
+                # TCP connection successful, verify HTTP endpoint
+                try:
+                    response = httpx.get(f"{server_url}/health", timeout=1.0)
+                    if response.status_code == 200:
+                        ready = True
+                        break
+                except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)